docs/datamodel/links.rst

```diff
@@ -134,6 +138,29 @@ Parameters:
 :eschema:synopsis:`<constraint-declarations>`
     :ref:`Constraint <ref_datamodel_constraints>` declarations.
 
+:eschema:synopsis:`on target delete`
+    On target delete options cover the situation when the target
+    object of a link is deleted without explicitly updating the link.
+
+:eschema:synopsis:`restrict`
+    Prohibit deleting the link target as long as the source object exists.
+    This is the default behavior.
+
+:eschema:synopsis:`allow`
+    Allow dropping the connection between the source and target when
+    the target is deleted.
+
+:eschema:synopsis:`delete source`
+    Delete the source object if any link target is deleted. This means
+    that for ``multi`` links the source object will be deleted
+    if even one of the link targets is deleted (e.g. automatically
+    dissolving a team when all team members are critical and one has
+    been deleted).
+
+:eschema:synopsis:`deferred restrict`
+    Similar to restrict, but no action is taken if the link is
```

Comment:
Better wording: "Same as `restrict`, but the check is performed at the end of transaction instead of immediately".

Reply:
Fixed